teaser/logic/buildingobjects/calculation/two_element.py

```diff
@@ -402,8 +402,9 @@ def calc_attributes(self):
                           ", please be careful with results. In addition " +
                           "this might lead to RunTimeErrors")
         self._sum_outer_wall_elements()
-        if len(self.thermal_zone.inner_walls) < 1:
-            warnings.warn('For thermal zone ' + self.thermal_zone.name,
+        if len(self.thermal_zone.inner_walls + self.thermal_zone.floors +
```

Comment:
I fixed both points. I didn't know why internal walls have been treated differently, it seemed to work. Anyway, they are now treated similar and it still works. :-)
For the second point, this forced a lot of additional checks that are all added now.